src/docfx/lib/HtmlUtility.cs

```diff
@@ -402,42 +404,38 @@ private static bool IsLink(ref HtmlToken token, in HtmlAttribute attribute)
         private static int CountWordInText(ReadOnlySpan<char> text)
         {
             var total = 0;
-            var isEnglishWord = false;
+            var word = false;
 
             foreach (var ch in text)
             {
                 if (IsCJKChar(ch))
                 {
                     total++;
 
-                    if (isEnglishWord)
+                    if (word)
                     {
-                        isEnglishWord = false;
+                        word = false;
                         total++;
                     }
                 }
                 else
                 {
-                    if (ch == ' ' || ch == '\t' || ch == '\n' || ch == '\r')
+                    if (s_wordSeparatorChars.Any(i => i == ch))
```

Comment:
Revert this change because `Enumerable.Any` allocates memory and is much slower than the old version. Same as line 431.

Please also use the same technique to implement `IsCJKChar` by inlining the range check directly in line 411